docs/features/middlewares.rst

```diff
@@ -6,8 +6,73 @@ Middlewares
 Description
 -----------
 
+Middleware functions in Django allow you to process requests and responses globally before they reach the view or after they leave the view. These middleware functions are wrapped with the sync_and_async_middleware decorator, indicating that they are compatible with both synchronous and asynchronous code.
+
```

Comment:
Middlewares in dsg are made to be almost compatible with django middlewares, the only difference is the argument being of type  `django_socio_grpc.request_transformer.GRPCRequestContainer` 
you can add a link to django doc regarding middlewares